config/routes.rb

```diff
@@ -102,11 +102,14 @@ def matches?(request)
         get :per_week
       end
     end
+    resources :suggested_councillors, only:[:new, :create] 
     collection do
       get :test_feed
     end
   end
 
+  resources :councillors, only: [:new, :create]
```

Comment:
this is the time when I was trying to add a column in councillor table. not relevant now.